cmd/kyma/install/cmd.go

```diff
@@ -738,9 +738,15 @@ func (cmd *command) printSummary() error {
 		return err
 	}
 
+	var consoleURL string
 	vs, err := cmd.K8s.Istio().NetworkingV1alpha3().VirtualServices("kyma-system").Get("core-console", metav1.GetOptions{})
-	if err != nil {
+	switch {
+	case apiErrors.IsNotFound(err):
+		consoleURL = "not found"
+	case err != nil:
 		return err
+	case vs != nil:
```

Comment:
what if vs.Spec.Hosts is empty? probably there will be panic. What message should be displayed then, "no information available"?